tensorflow_federated/python/aggregators/quantile_estimation.py

```diff
@@ -106,7 +106,7 @@ def no_noise(cls,
   def __init__(
       self,
       quantile_estimator_query: tfp.QuantileEstimatorQuery,
-      record_aggregation_factory: factory.UnweightedAggregationFactory = None):
+      record_aggregation_factory: factory.UnweightedAggregationFactory):
```

Comment:
Actually, I think this type annotation should be updated to be `Optional` (e.g. Optional[factory.UnweightedAggregationFactory]).

This requires `from typing import Optional`